components/brave_vpn/common/BUILD.gn

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/components/brave_vpn/buildflags/buildflags.gni")
+
+source_set("common") {
+  sources = [
+    "brave_vpn_constants.h",
+    "brave_vpn_data_types.h",
+    "brave_vpn_utils.cc",
+    "brave_vpn_utils.h",
+    "features.cc",
+    "features.h",
+    "pref_names.cc",
+    "pref_names.h",
+  ]
+  deps = [
+    "//brave/components/p3a_utils",
+    "//brave/components/skus/browser",
+    "//brave/components/skus/common",
+    "//build:buildflag_header_h",
+    "//components/pref_registry",
+    "//components/prefs",
+    "//net",
+  ]
+}
+
+source_set("unit_tests") {
+  if (enable_brave_vpn) {
```

Comment:
I would drop this in favor of `assert(enable_brave_vpn)` for the entire file which will ensure that the target is properly guarded when added as a dep

Reply:
done